src/pieces/memberHandler.ts

```diff
@@ -35,6 +39,24 @@ async function memberUpdate(oldMember: GuildMember | PartialGuildMember, newMemb
 	}
 }
 
+async function memberRemove(member: GuildMember | PartialGuildMember): Promise<void> {
+	if (member.guild.id !== GUILDS.MAIN) return;
+	if (member.partial) {
+		await member.fetch();
```

Comment:
Since this is a remove event, the full member can't be fetched as it no longer exists.